diesel/src/migrations/mod.rs

```diff
@@ -163,7 +163,7 @@ pub fn setup_database<Conn: Connection>(conn: &Conn) -> QueryResult<usize> {
 fn create_schema_migrations_table_if_needed<Conn: Connection>(conn: &Conn) -> QueryResult<usize> {
     conn.silence_notices(|| {
         conn.execute("CREATE TABLE IF NOT EXISTS __diesel_schema_migrations (
-            version VARCHAR PRIMARY KEY NOT NULL,
+            version VARCHAR(255) PRIMARY KEY NOT NULL,
```

Comment:
I have to put something here. MySQL doesn't allow `TEXT` PKs (because reasons I guess?). 255 has always been the go-to size since the length fits in one byte. Are you suggesting we go up a power of two?